db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
```

Comment:
- [ ] Please create 5 different messages here to store them in the database so that when the app opens and invokes the API finds a random greeting message to render. The seeds file is the place where you can create them.